README.md

```diff
@@ -40,11 +40,11 @@ Clone the github repository:
 
     git clone https://github.com/angular/protractor.git
     cd protractor
+
+Install all the dependencies in the folder:
+
     npm install
-    ./bin/webdriver-manager update
-    cd website
-    npm install
-    cd ..
+
```

Comment:
These instructions should be left in. These instructions are important for contributors.

We could do an `npm run install_website` here instead.